Examples.scala

```diff
@@ -340,4 +341,11 @@ object ex5 {
     }
   }
 
+  def sleep(f: FiniteDuration): IO[Unit] = IO.async { cb =>
+    val rightUnit = Right(())
+    sec.schedule(new Runnable {
+      override def run(): Unit = cb(rightUnit)
+    }, f._1, f._2)
+  }
```

Comment:
This is not correct as you would then run all the subsequent code on the ScheduledPool. Look at the slides for what you should do instead

Reply:
Thanks for the tip, I have fixed it